sdk/java-sdk-protobuf/src/main/scala/kalix/javasdk/impl/logging/KalixLogbackConfigurator.scala

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2021 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package kalix.javasdk.impl.logging
+
+import java.io.File
+
+import ch.qos.logback.classic.LoggerContext
+import ch.qos.logback.classic.spi.Configurator
+import ch.qos.logback.core.spi.ContextAwareBase
+
+class KalixLogbackConfigurator extends ContextAwareBase with Configurator {
```

Comment:
now I understand, I thought we set the System.property from `main`, but I guess this is guaranteed to be loaded first before logback is initialized